.gitignore

```diff
@@ -1,5 +1,6 @@
 # Ignore all compiled files
 #--------------------------
+sorc/nceplibs-ncio/
```

Comment:
why is this relevant to this PR?

Reply:
This is a directory of some compiled objects? no?

Reply:
Leave it out. PR #375 eliminates the hack that creates this anyway.